source/fundamentals/stable-api.txt

```diff
@@ -0,0 +1,130 @@
+.. _csharp-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. note::
+
+   The {+stable-api+} feature requires MongoDB Server 5.0 or later.
+
+   You should use the {+stable-api+} feature only if all the MongoDB
+   servers you're connecting to support this feature.
+
+Overview
+--------
+
+When a MongoDB server runs an operation, it uses the version of the operation defined by
+the current *API version*. When you update either your driver or server, it changes 
```

Comment:
```suggestion
The {+stable-api+} feature forces the server to run operations with behaviors compatible with the *API version* you specify. When you update either your driver or server, it changes 
```

Reply:
I was also intentionally trying something different here--starting with a "hook," generally used in training to get people to care about what they're reading, instead of jumping straight into the explanation. Maybe that's not the format we want!